help/en/html/tools/logixng/reference/WebRequestExample/WebRequest.xml

```diff
@@ -70,7 +70,7 @@ This file is created by jmri.jmrit.logixng.actions.WebRequestTest
   <signalmastlogics class="jmri.managers.configurexml.DefaultSignalMastLogicManagerXml">
     <logicDelay>500</logicDelay>
   </signalmastlogics>
-  <timebase class="jmri.jmrit.simpleclock.configurexml.SimpleTimebaseXml" time="Wed Aug 23 21:55:53 CEST 2023" rate="1.0" startrate="1.0" run="yes" master="yes" sync="no" correct="no" display="no" startstopped="no" startrunning="yes" startsettime="no" startclockoption="0" showbutton="no" startsetrate="yes" />
```

Comment:
@danielb987 The changes to this file are just the timestamps. Is this OK?

Reply:
Yes, these changes are fine. This file is automatically created by the LogixNG test suite.